src/modules/dashboard/sagas/motions/paymentMotion.ts

```diff
@@ -138,33 +142,45 @@ function* createPaymentMotion({
       ],
     );
 
-    const { skillId } = yield call([client, client.getDomain], ROOT_DOMAIN_ID);
+    const { skillId } = yield call(
+      [colonyClient, colonyClient.getDomain],
+      domainId,
+    );
 
     const { key, value, branchMask, siblings } = yield call(
-      client.getReputation,
+      colonyClient.getReputation,
       skillId,
       AddressZero,
     );
 
     txChannel = yield call(getTxChannel, metaId);
 
     // setup batch ids and channels
-    const batchKey = 'createRootMotion';
+    const batchKey = 'createMotion';
 
     const {
-      createRootMotion,
+      createMotion,
       annotatePaymentMotion,
     } = yield createTransactionChannels(metaId, [
-      'createRootMotion',
+      'createMotion',
       'annotatePaymentMotion',
     ]);
 
     // create transactions
-    yield fork(createTransaction, createRootMotion.id, {
+    yield fork(createTransaction, createMotion.id, {
       context: ClientType.VotingReputationClient,
-      methodName: 'createRootMotion',
+      methodName: 'createMotion',
       identifier: colonyAddress,
-      params: [AddressZero, encodedAction, key, value, branchMask, siblings],
+      params: [
+        domainId,
+        childSkillIndex,
+        client.address,
+        encodedAction,
+        key,
+        value,
+        branchMask,
+        siblings,
+      ],
```

Comment:
When trying to create this payment in a child domain, this errors out:

![Screenshot from 2021-05-18 00-04-43](https://user-images.githubusercontent.com/1193222/118557135-72c89b80-b76d-11eb-83c0-5178eeb382f7.png)

I'm assuming it doesn't like the `childSkillIndex` value, but this is just a wild guess. I think you might need to bring in Alex to help you on this.

Reply:
Do you have reputation in the domain where you are creating the motion? I think this is the reason why.
I can't replicate. I've done several tests and it works every time except when I don't have reputations in that domain.

But it's a good catch because I forgot to add the reputation check for domain reputation.

Reply:
i've fixed it the reputation check and am pushing the changes.